Bender.yml

```
@@ -33,6 +33,7 @@ dependencies:
   snax-gemm:          { git: "https://github.com/KULeuven-MICAS/snax-gemm", rev: main }
   snax-postprocessing-simd: { git: "https://github.com/KULeuven-MICAS/snax-postprocessing-simd", rev: main }
   hypercorex:         { git: "https://github.com/KULeuven-MICAS/hypercorex.git", rev: main }
+  dimc:               { git: "https://github.com/mace1997/dimc.git", rev: main }
```

Can we please include this from KULeuven-MICAS instead?
```suggestion
  dimc:               { git: "https://github.com/KULeuven-MICAS/dimc.git", rev: main }
```

---

Oh yeah, wait, we'll fix it on Mace's side.

---

:pray: